packages/contracts/contracts/tcr/PLCRVoting.sol

```diff
@@ -14,25 +15,29 @@ contract PLCRVoting {
   // EVENTS:
   // ============
 
-  event VoteCommitted(address indexed voter, uint indexed pollID, uint numTokens);
-  event VoteRevealed(address indexed voter, uint indexed pollID, uint numTokens, uint choice);
-  event PollCreated(uint voteQuorum, uint commitDuration, uint revealDuration, uint indexed pollID);
-  event VotingRightsGranted(address indexed voter, uint numTokens);
-  event VotingRightsWithdrawn(address indexed voter, uint numTokens);
+  event _VoteCommitted(uint indexed pollID, uint numTokens, address indexed voter);
```

Comment:
is the underscore here just to be consistent with upstream, or for some other reason?